src/hydra-eval-jobs/Makefile.am

```diff
@@ -1,5 +1,5 @@
 bin_PROGRAMS = hydra-eval-jobs
 
 hydra_eval_jobs_SOURCES = hydra-eval-jobs.cc
-hydra_eval_jobs_LDADD = $(NIX_LIBS) -lnixcmd
+hydra_eval_jobs_LDADD = $(NIX_LIBS) -lnixcmd -lnixflake
```

Comment:
I am now thinking about making hydra-eval-jobs it's own derivation so that we can split the nix update over multiple pull requests.

Reply:
That would be good for deduplicating with nix-eval-jobs too 

https://github.com/NixOS/hydra/issues/1315

Also hope we can convert hydra to Meson and then use subprojects to still support incremental builds of everything.